docs/quickstart.md

```diff
@@ -1,1064 +1,9 @@
-          - envVarSchema:
-              subPath:
-                required: true
-            id: 0
-            rule: Path(`/{{.Vars.subPath}}/hello`)
-          type: KubeFox
-      containerRegistry: localhost/kubefox
-    status:
-      conditions:
-      - lastTransitionTime: "1970-01-01T00:00:00Z"
-        message: Component Deployments have minimum required Pods available.
-        observedGeneration: 1
-        reason: ComponentsAvailable
-        status: "True"
-        type: Available
-      - lastTransitionTime: "1970-01-01T00:00:00Z"
-        message: Component Deployments completed successfully.
-        observedGeneration: 1
-        reason: ComponentsDeployed
-        status: "False"
-        type: Progressing
-    ```
-
-Inspect what's running on Kubernetes.
-
-```{ .shell .copy }
-kubectl get pods --namespace kubefox-demo
-```
-
-??? example "Output"
-
-    ```text
-    NAME                                           READY   STATUS    RESTARTS   AGE
-    demo-broker-grkcn                              1/1     Running   0          12s
-    demo-httpsrv-7d8d946c57-rlt55                  1/1     Running   0          10s
-    demo-nats-0                                    1/1     Running   0          18s
-    hello-world-backend-3fbfcb2-577868c97b-29q2k   1/1     Running   0          2s
-    hello-world-frontend-98c75bd-65fb98f59d-ll4sf  1/1     Running   0          2s
-    ```
-
-The Pods for two Components you deployed were created, `backend` and `frontend`.
-The `broker`, `httpsrv`, and `nats` Pods are part of the KubeFox Platform
-initiated by the Operator during Platform creation.
-
-Typically, connections to KubeFox Apps are made through a public-facing load
-balancer. For the simplicity of this guide use Fox to create a local proxy
-instead. In a new terminal run the following command.
-
-```{ .shell .copy }
-fox proxy 8080
-```
-
-??? info "macOS Network Warning"
-
-    <figure markdown>
-      ![macosx-warning](images/fox-mac-net-warn.png)
-    </figure>
-
-    If you are using macOS you might notice this dialog popup when you start the
-    proxy. This is expected as Fox starts a local HTTP server. The server is
-    bound to the `localhost` interface and is only accessible from your
-    workstation. Please press `Allow` to continue.
-
-??? example "Output"
-
-    ```text
-    HTTP proxy started on http://127.0.0.1:8080
-    ```
-
-Now, back in the original terminal, test the AppDeployment. KubeFox won't route
-requests to the App until it's released, but you can still test AppDeployments
-by manually providing context. KubeFox needs two pieces of information to route
-an event, the AppDeployment to use and the Virtual Environments to inject. These
-can be passed as headers or query parameters for HTTP requests.
-
-```{ .shell .copy }
-curl "http://localhost:8080/qa/hello?kf-dep=hello-world-main&kf-ve=qa"
-```
-
-??? example "Output"
+<meta http-equiv="refresh" content="0; url=getting_started/quickstart.html" />
```

Comment:
@xadhatter This appears to work as expected when I deploy the mkdocs to my local machine - In lieu of the automatic redirect working I added in a manual button they can click to go to the new page.